pull_request.md

```diff
@@ -0,0 +1,39 @@
+This is a:
+
+- [x] ✨ **New feature** - new behaviour has been implemented
+- [ ] 🐛 **Bug fix** - existing behaviour has been made to behave
+- [ ] ♻️ **Refactor** - the behaviour has not changed, just the implementation
+- [ ] ✅ **Test backfill** - tests for existing behaviour were added but the behaviour itself hasn't changed
+- [ ] ⚙️ **Chore** - maintenance task, behaviour and implementation haven't changed
+
+### Description
+
+- **Purpose** - Adds a resource details page accessible from the resource list:
+
+  ![Screenshot 2023-08-15 at 11 40 44](https://github.com/CodeYourFuture/tech-products-demo/assets/785939/b51b0cf8-c8b1-4a5a-84b2-825b071ce3ba)
+
+- **How to check** -
+  - Ensure there are some published resources
+  - Visit the home page
+  - Click on a resource title
+  - See resource details per screenshot above
+
+### Links
+
+- CodeYourFuture/tech-products-demo#18
+
+### Author checklist
+
+<!-- All PRs -->
+
+- [ ] I have written a title that reflects the relevant ticket
+- [ ] I have written a description that says what the PR does and how to validate it
+- [ ] I have linked to the project board ticket (and any related PRs/issues) in the Links section
+- [ ] I have added a link to this PR to the ticket
+- [ ] I have made the PR to `main` from a branch named `<category>/<name>`, e.g. `feature/edit-spaceships` or `bugfix/restore-oxygen`
+- [ ] I have manually tested that the app still works correctly
+- [ ] I have requested reviewers here and in my team chat channel
+<!-- depending on the task, the following may be optional -->
+- [ ] I have spoken with my PM or TL about any parts of this task that may have become out-of-scope, or any additional improvements that I now realise may benefit my project
+- [ ] I have added tests, or new tests were not required
+- [ ] I have updated any documentation (e.g. diagrams, schemas), or documentation updates were not required
```

Comment:
💡 Considering the requirements of this task, it appears unnecessary. However, if it provides a sample template for the pull request to be raised, then I would suggest adding it to the main branch.

Reply:
❌ The purpose of this file, per the task description, is to use it _as the description_ for this PR. The template already exists in `.github/PULL_REQUEST_TEMPLATE.md`, this is the same template _plus relevant content_ for the changes being proposed.

Reply:
💡 It is understandable, but basically, for this specific task, developers to copy and create a new pull request likely to this very one, which would be placed in a different folder and then direct developers find it.

Reply:
❓ I'm not sure what you're suggesting here, I think the task description already _does_ direct developers to find it, although maybe not clearly enough. Perhaps you could make a pull request to the upstream repo reflecting how you think it would be better laid out and described? I think it's off-topic for this PR review though.